src/stashmanager.hpp

```diff
@@ -0,0 +1,16 @@
+#pragama once
+#include "HashTables.hpp"
+class StashManager {
+  Hashtable<Stash*> stashes;
```

Comment:
Recuerda que la clase se llama `HashTable`.

```suggestion
  HashTable<Stash*> stashes;
```

Reply:
La clase si se llama "HashTable" pero el nombre del hpp esta puesto como "HashTables.hpp" deberia cambiarle el nombre al header file o solo a esa linea 4 ?